addons/website/static/src/js/backend/res_config_settings.js

```diff
@@ -33,4 +39,49 @@ BaseSettingController.include({
     },
 });
 
+const WebsiteCookiesbarField = FieldBoolean.extend({
+    _onChange: function () {
+        const checked = this.$input[0].checked;
+        if (!checked) {
+            return this._setValue(checked);
+        }
+
+        let message = $('<main/>', {
+            role: 'alert',
+        }).html(
+            $('<p/>').html(
+                _t("<b>Cookie bars are bad for the experience</b> of your visitors. We believe you should not annoy your visitors with such a practice, unless you absolutly need it.")
+            )
+        ).append(_.str.sprintf(
+            _t("For session cookies, authentification and analytics, <b>you do not need to ask for the consent</b>, provided that your analytics is anonymized, which should be the case if you use Google Analytics. (%s, adopted in June 2012)"),
+            "<a target='_blank' href='https://ec.europa.eu/justice/article-29/documentation/opinion-recommendation/files/2012/wp194_en.pdf'>cf wp194 of Art 29</a>"
```

Comment:
Approved by FP? Same for btns?

Reply:
Done by FP actually, yeah..

Reply:
Last time I checked, GA was not anonymous by default, [you need to opt-out of the storage of the last byte of the IP](https://support.google.com/analytics/answer/2763052?hl=en) - it's worth mentioning, isn't it?

Also why link to the PDF that could 404 at some point, rather than simply quote "*see e.g. Opinion 04/2012 on Cookie Consent Exemption by the EU Art.29 WP*", which will be easy to google for anyone who cares.

If we keep the harcoded link, making it "nofollow noopener noreferrer" makes sense I think, because it's external.

Reply:
I'd rather have no popup on cookies feature activation tbh.
But you are right about the PDF link, unless we host it on our own server, we can't ensure it will be available forever.

Reply:
Unless I missed something, current text still seems to imply that GA is anonymous by default?

Reply:
Didn't process that part yet, should be good now